src/Libraries/RevitNodes/Filter/OverrideGraphicSettings.cs

```diff
@@ -67,8 +67,8 @@ internal OverrideGraphicSettings(Autodesk.Revit.DB.OverrideGraphicSettings inter
             Autodesk.Revit.DB.OverrideGraphicSettings filterSettings = new Autodesk.Revit.DB.OverrideGraphicSettings();
 
             // Apply Colors
-            if (cutFillColor != null) filterSettings.SetCutFillColor(ToRevitColor(cutFillColor));
-            if (projectionFillColor != null) filterSettings.SetProjectionFillColor(ToRevitColor(projectionFillColor));
+            if (cutFillColor != null) filterSettings.SetCutForegroundPatternColor(ToRevitColor(cutFillColor));
+            if (projectionFillColor != null) filterSettings.SetSurfaceForegroundPatternColor(ToRevitColor(projectionFillColor));
             if (cutLineColor != null) filterSettings.SetCutLineColor(ToRevitColor(cutLineColor));
```

Comment:
please add comments why we did these changes? like the old methods are obsoleted and the new methods are suggested by the documentation. Did you test them - manually or via RTFRevit?

Reply:
@philxia1 I have added some comments to the replaced.